examples/image_ocr.py

```diff
@@ -0,0 +1,433 @@
+# This example uses a convolutional stack followed by a recurrent stack
+# and a CTC logloss function to perform optical character recognition
+# of generated text images. I have no evidence of whether it actually 
+# learns general shapes of text, or just is able to recognize all
+# the different fonts thrown at it...the purpose is more to demonstrate CTC
+# inside of Keras.  Note that the font list may need to be updated
+# for the particular OS in use. 
+# 
+# This starts off with easy 5 letter words. After 10 or so epochs, CTC  
+# learn translataional invariance, so longer words and groups of words 
+# with spaces are gradually fed in. 
```

Comment:
Please describe the accuracy (e.g. average edit distance) that you are seeing after x epochs, so users know what to expect when running the script.